app/models/components/course/course_ability_component.rb

```diff
@@ -37,6 +38,11 @@ def allow_staff_show_course_users
     can :show_users, Course, id: course.id
   end
 
+  def allow_staff_read_all_experience_points
```

Comment:
You try to masquerade as a student, run this in `http://lvh.me:8080/courses/{course_number}/experience_points_records?format=json` and see if your backend authorization works properly